.vib/suitecrm/goss/vars.yaml

```diff
@@ -0,0 +1,32 @@
+binaries:
+  - apachectl
+  - mariadb
+  - php
+  - render-template
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/suitecrm
+      - /opt/bitnami/suitecrm
+      - /opt/bitnami/suitecrm/public
+      - /opt/bitnami/suitecrm/public/legacy
+      - /opt/bitnami/suitecrm/public/legacy/cache
+      - /opt/bitnami/suitecrm/public/legacy/custom
+      - /opt/bitnami/suitecrm/public/legacy/data
+      - /opt/bitnami/suitecrm/public/legacy/modules
+      - /opt/bitnami/suitecrm/public/legacy/upload
```

Comment:
I'm not sure where are we creating and/or modifying these directories in our logic. Keep in mind we should abstain from checking the filesystem not modified prior to this test suite execution. That is, we should focus on checking the logic executed be it in our compilation and postunpack stages.

Reply:
Well, this was based on the tests from our previous test system. But I will remove most of them since we are already granting W privileges to most files thanks to this line:

```
for dir in "$SUITECRM_BASE_DIR" "$SUITECRM_VOLUME_DIR" "${SUITECRM_BASE_DIR}/tmp"; do
    ...
    configure_permissions_ownership "$dir" -d "775" -f "664" -u "$WEB_SERVER_DAEMON_USER" -g "root"
```

But I will keep the one for `public` since it is a sub-folder and we want to check its permissions.